src/api/layer/bankTransactions.ts

```diff
@@ -132,15 +132,15 @@ export const getBankTransactionDocument = get<{
 )
 
 export const archiveBankTransactionDocument = post<{
-  data: {}
+  data: Record<never, never>
   errors: unknown
 }>(
   ({ businessId, bankTransactionId, documentId }) =>
     `/v1/businesses/${businessId}/bank-transactions/${bankTransactionId}/documents/${documentId}/archive`,
 )
 
 export const uploadBankTransactionDocument =
-  (baseUrl: string, accessToken: string) =>
+  (baseUrl: string, accessToken?: string) =>
```

Comment:
should this function not be called if accessToken is undefined? same questien in tasks.ts